library/src/main/java/com/novoda/downloadmanager/PartialDownloadMigrationExtractor.java

```diff
@@ -7,54 +7,50 @@
 import java.util.List;
 import java.util.Set;
 
-class PartialDownloadMigrationExtractor {
+public class PartialDownloadMigrationExtractor {
```

Comment:
So that it can be used by the activity (will move module later).

Reply:
+1 for moving into the demo.

Should this be called a `VersionOnePartialMigrationExtractor`

Reply:
More of a `VersionOnePartialDownloadBatchesExtractor` I guess 😄